src/sentry/notifications/utils/__init__.py

```diff
@@ -450,14 +450,15 @@ def get_performance_issue_alert_subtitle(event: Event) -> str:
 def get_notification_group_title(
     group: Group, event: Event | GroupEvent, max_length: int = 255, **kwargs: str
 ) -> str:
-    if group.issue_category == GroupCategory.PERFORMANCE:
+    # Verify that we should prioritise `issue_title` over the description for perf issues
```

Comment:
yeah group.title is the same as issue_type.description so we should be good https://github.com/getsentry/sentry/blob/master/src/sentry/utils/performance_issues/performance_problem.py#L45